src/components/AppHeader/AppHeader.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from 'react';
+import styles from './AppHeader.module.css';
+import logoIcon from 'src/assets/icons/logo-icon.svg';
+import cartIcon from 'src/assets/icons/cart-icon.svg';
+
+const AppHeader = () => {
+    const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+    const navItems = [
```

Comment:
It's better to move such object into the separate file to have possibility easy replase with other data source

Reply:
Moved data to separate files in data folder.